Doc/library/tracemalloc.rst

```diff
@@ -30,8 +30,118 @@ frame (1 frame). To store 25 frames at startup: set the
 :option:`-X` ``tracemalloc=25`` command line option.
 
 
-Examples
---------
+Quickstart Tutorial
+-------------------
+
+Essential :mod:`tracemalloc` API
```

Comment:
I don't think that this section is enough. When I learn a new module, I prefer just to copy/paste examples without trying to understand them. Examples are short enough.